static/src/javascripts/projects/commercial/modules/dfp/on-slot-viewable.js

```diff
@@ -5,29 +5,19 @@ import { Advert } from 'commercial/modules/dfp/Advert';
 import { getAdvertById } from 'commercial/modules/dfp/get-advert-by-id';
 import { enableLazyLoad } from 'commercial/modules/dfp/lazy-load';
 
-const getViewabilityThreshold = (advert: Advert): ?number => {
-    const refreshThresholdMsDefault = 30000;
-    const refreshThresholdMsVideo = 90000;
-
+const shouldRefresh = (advert: Advert): boolean => {
     const fluidSize = '0,0';
-
-    const sizeString = advert.size && advert.size.toString();
     const videoSizes = ['620,1', '620,350'];
+    const sizeString = advert.size && advert.size.toString();
 
-    if (sizeString === fluidSize) {
-        return;
-    } else if (videoSizes.includes(sizeString)) {
-        return refreshThresholdMsVideo;
-    }
-
-    return refreshThresholdMsDefault;
+    return sizeString === fluidSize || videoSizes.includes(sizeString);
 };
 
 export const onSlotViewable = (event: ImpressionViewableEvent): void => {
     const advert: ?Advert = getAdvertById(event.slot.getSlotElementId());
-    const viewabilityThresholdMs = advert && getViewabilityThreshold(advert);
+    const viewabilityThresholdMs = 30000;
 
-    if (advert && viewabilityThresholdMs) {
+    if (advert && shouldRefresh(advert)) {
```

Comment:
does that mean if it is a fluid or video, it will set the timeout? is the logic the right way round?

Reply:
It absolutely isn't - I hadn't pushed my other change up! i.e. a `!`